modules/@apostrophecms/modal/ui/apos/mixins/AposEditorMixin.js

```diff
@@ -36,6 +36,8 @@ export default {
           field.readOnly = true;
         }
       }
+      // Trash UI is handled via action buttons
+      schema = schema.filter(field => field.name !== 'trash');
```

Comment:
Should trash still be a field in the doc-type schema? 🤔

Reply:
Yes, for pieces it should, because it is still handled that way on the back end, e.g. via PATCH of a normally converted schema field. So the front end presenting it in a particular way is the front end's business but it's still a schema field.